config/examples/Creality/CR-10S/BigTreeTech SKR Mini E3 3.0/Configuration_adv.h

```diff
@@ -575,7 +575,7 @@
  */
 #define USE_CONTROLLER_FAN
 #if ENABLED(USE_CONTROLLER_FAN)
-  //#define CONTROLLER_FAN_PIN -1           // Set a custom pin for the controller fan
+  #define CONTROLLER_FAN_PIN FAN1_PIN       // Set a custom pin for the controller fan
```

Comment:
For the SKR Mini E3 V3-based configs, you’d typically set this to `FAN2_PIN` and use `FAN1_PIN` for `E0_AUTO_FAN_PIN`.

Reply:
Yes I see... though there is of course one eg that doesn't (ender 3 v2)